components/esp32_camera.rst

```diff
@@ -489,13 +489,15 @@ Configuration examples
           url: https://github.com/MichaKersloot/esphome_custom_components
         components: [ esp32_camera ]
 
+    i2c:
+      - id: camera_i2c
+        sda: GPIO4
+        scl: GPIO5
     esp32_camera:
       external_clock:
         pin: GPIO15
         frequency: 20MHz
-      i2c_pins:
```

Comment:
This one was actually left in on purpose due to the external component
<https://github.com/MichaKersloot/esphome_custom_components/issues/2>

Reply:
If you want to implement the changes as specified in that issue, I'm sure everyone will be thankful =)